cuda/test/matrix/dense_kernels.cpp

```diff
@@ -31,15 +31,21 @@ ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ******************************<GINKGO LICENSE>*******************************/
 
+#include <ginkgo/core/base/array.hpp>
+#include <ginkgo/core/matrix/coo.hpp>
+#include <ginkgo/core/matrix/csr.hpp>
 #include <ginkgo/core/matrix/dense.hpp>
+#include <ginkgo/core/matrix/ell.hpp>
+
+
+#include "core/matrix/dense_kernels.hpp"
 
 
 #include <gtest/gtest.h>
 
 
 #include <random>
 
-
```

Comment:
Always 2 newlines to separate different groups of headers (here standard header and public ginkgo header)